prediction-insights.ipynb

```diff
@@ -0,0 +1,774 @@
+    "The predicted value(s) are accessed using the `action` key, while the explanations can be accessed using the\n",
+    "`details` key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['influential_cases',\n",
+       " 'feature_contributions',\n",
+       " 'action_features',\n",
+       " 'feature_residuals',\n",
+       " 'local_case_feature_residual_convictions',\n",
+       " 'similarity_conviction',\n",
+       " 'categorical_action_probabilities',\n",
+       " 'directional_feature_contributions']"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "list(reaction[\"details\"].keys())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>target</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   target\n",
+       "0       1"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reaction[\"action\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For categorical action features, the prediction can be further understood with the `categorical_action_probabilities`\n",
+    "detail."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'target': {'1': 0.8815330011979926, '0': 0.11846699880200744}}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reaction[\"details\"][\"categorical_action_probabilities\"][0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Insight 2: Attribution\n",
+    "\n",
+    "Howso provides complete attribution for any and all predictions. To determine which cases were used to make a prediction, use the `influential_cases` detail."
```

Comment:
Let's have a discussion of what all the types of attribution we can get are: 
Influential cases - these values impacted the decision, so you can see _why_ the decision was made
Most similar cases - these are the original cases that have the most similar features to the predicted case; this enables you to understand how your prediction case compares to other original cases the trainee is familiar with
Boundary cases - similar to counterfactuals, these tell you the original cases that had similar features/traits to the predicted case, but came to a different outcome...

etc